test/routes.test.js

```diff
@@ -1,114 +1,62 @@
-					data: { owner: { login: "umbrella-corp" } }
-				})
-			},
-			// Assuming we're not going to paginate in these tests.
-			paginate: async (octokitCall, process) => {
-				return Promise.resolve(process(await octokitCall));
-			}
-		};
-
-		// Passes the mocked out GitHub API into our app instance.
-		app.auth = () => Promise.resolve(github);
-
-		// Register our routes with the empty application.
 		await routes(app);
-
-		// Setup an express object that we can test.
-		server.use(app.router);
 	});
 
-	test("reject unauthorised request when a bearer token is loaded", async () => {
+	test("rejects an incorrect Bearer token", async () => {
 		await request(server)
 			.get("/tako/repositories")
 			.set("Accept", "application/json")
-			.expect(401);
-
-		await request(server)
-			.get("/tako/repositories?topic=express")
-			.set("Accept", "application/json")
+			.set("Authorization", "Bearer incorrect-bearer-token")
 			.expect(401);
 	});
 
-	test("accepts authorised request when a bearer token is loaded", async () => {
+	test("accepts a correct Bearer token", async () => {
 		await request(server)
 			.get("/tako/repositories")
 			.set("Accept", "application/json")
-			.set("Authorization", "Bearer hunter2")
-			.expect(200);
-
-		await request(server)
-			.get("/tako/repositories?topic=express")
-			.set("Accept", "application/json")
-			.set("Authorization", "Bearer hunter2")
+			.set("Authorization", "Bearer correct-bearer-token")
 			.expect(200);
 	});
 
 	test("noCache sets a cache-control header with a value of max-age=0", async () => {
 		await request(server)
 			.get("/tako/repositories")
 			.set("Accept", "application/json")
-			.set("Authorization", "Bearer hunter2")
-			.expect("Cache-Control", "max-age=0, no-cache");
-
-		await request(server)
-			.get("/tako/repositories?topic=express")
-			.set("Accept", "application/json")
-			.set("Authorization", "Bearer hunter2")
+			.set("Authorization", "Bearer correct-bearer-token")
 			.expect("Cache-Control", "max-age=0, no-cache");
 	});
 
 	test("/tako/repositories reponds ok", async () => {
 		await request(server)
 			.get("/tako/repositories")
 			.set("Accept", "application/json")
-			.set("Authorization", "Bearer hunter2")
+			.set("Authorization", "Bearer correct-bearer-token")
 			.expect("Content-Type", "application/json; charset=utf-8")
 			.expect(200);
 	});
 
-	test("/tako/repositories?topic=express responds ok", async () => {
+	test("/tako/repositories?topic=foo responds ok", async () => {
 		await request(server)
-			.get("/tako/repositories?topic=express")
+			.get("/tako/repositories?topic=foo")
 			.set("Accept", "application/json")
-			.set("Authorization", "Bearer hunter2")
+			.set("Authorization", "Bearer correct-bearer-token")
 			.expect("Content-Type", "application/json; charset=utf-8")
 			.expect(200);
 	});
-
-	test("/tako/repositories?topic=fake-github-auth-erroring responds with an error", async () => {
```

Comment:
This is covered in https://github.com/Financial-Times/tako/blob/master/test/initialise.test.js#L40